platform/accessibility/guidance/staging-review-processes.md

```diff
@@ -0,0 +1,51 @@
+  * [Test with 1 or 2 screen readers](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/508-accessibility-best-practices.md#keyboard-navigation) _(VoiceOver or NVDA)_
+* Provide documentation for your test cases using one of the templates:
+  * [TestRail accessibility test plan template](https://dsvavsp.testrail.io/index.php?/suites/view/14&group_by=cases:section_id&group_order=asc) **OR** 
+  * [Github accessibility test cases template](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/accessibility-test-cases-template.md)
+* Log any known issues using the [508 issue template](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=&labels=508%2FAccessibility&template=508-issue.md&title=)
+
+## Staging Review for Applications
+Accessibility specialist(s) will run through the the following automated and manual tests for rich applications. These include  React applications, forms, and anything else that isn't static HTML.
+
+* Axe browser plugin checks on a subset of pages or app states to ensure coverage
+* Zooming layouts to 400% on a subset of pages
+* Once-through keyboard navigation
+* 3 preferred screen reader and browser pairings. Pairings will be determined by analytics data and WebAIM’s [most recent screen reader survey](https://webaim.org/projects/screenreadersurvey8/#primary). The favored browser and screen reader pairings may evolve as Edge Chromium and Chrome assume more market share, but the current pairings include:
+  * JAWS on Windows7/10 using IE11 _(This is the VA 508 office's preferred test pairing)_
+  * NVDA on Windows 7/10 using Firefox
+  * VoiceOver on MacOS
+* End-to-end tests will be reviewed to ensure axe checks are run for primary and secondary (hidden) content
+
+## Staging Review for Static Content
+Accessibility specialist(s) will run through the the following automated and manual tests for static content. Static content includes pages generated from the content management system (CMS) or Markdown files.
+
+* Axe browser plugin checks on a subset of pages to ensure coverage
+* Zooming layouts to 400% on a subset of pages
+* Once-through keyboard navigation
+* 3 preferred screen reader and browser pairings. Pairings will be determined by analytics data and WebAIM’s [most recent screen reader survey](https://webaim.org/projects/screenreadersurvey8/#primary). The favored browser and screen reader pairings may evolve as Edge Chromium and Chrome assume more market share, but the current pairings include:
+  * JAWS on Windows7/10 using IE11 _(This is the VA 508 office's preferred test pairing)_
+  * NVDA on Windows 7/10 using Firefox
+  * VoiceOver on MacOS
+
+## Issue Gathering and Triage
+* Validate code early and often to help identify issues sooner than later
+* Request a spot check 1-2 sprints before your anticipated staging review
```

Comment:
what's a "spot-check"?

could this be handled in the same process as "giving us a heads up that the staging review will be needed in 1-2 sprints"?

Reply:
Spot checks are off-the-cuff reviews, that are much less formal than the staging review. They're meant as collab sessions to improve code while it's malleable. I'll clarify this point.